From 5d3d6c58c644a53e65a780fa26c043d0dac0ba28 Mon Sep 17 00:00:00 2001
From: Ky Nguyen <ky.nguyen.uj@renesas.com>
Date: Mon, 14 Dec 2020 17:22:54 +0700
Subject: [PATCH 022/135] arm64: dts: rzg2-smarc: Enable SCIF0 as default debug
 port.

This commit enables and set SCIF channel 0 as default debug port.

Signed-off-by: Ky Nguyen <ky.nguyen.uj@renesas.com>
Signed-off-by: Binh Nguyen <binh.nguyen.jz@renesas.com>
---
 arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi b/arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi
index 07cee80..14c0d33 100644
--- a/arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi
+++ b/arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi
@@ -5,7 +5,32 @@
  * Copyright (C) 2020 Renesas Electronics Corp.
  */
 
+#include <dt-bindings/pinctrl/pinctrl-rzg2l.h>
+
+/ {
+	aliases {
+		serial0 = &scif0;
+	};
+
+	chosen {
+		bootargs = "ignore_loglevel";
+		stdout-path = "serial0:115200n8";
+	};
+};
+
 &xinclk {
 	clock-frequency = <24000000>;
 };
 
+&pinctrl {
+	scif0_pins: scif0 {
+		groups = "scif0_data";
+		function = "scif0";
+	};
+};
+
+&scif0 {
+	pinctrl-0 = <&scif0_pins>;
+	pinctrl-names = "default";
+	status = "okay";
+};
-- 
2.7.4

